src/sdl/index.ts

```diff
@@ -252,6 +252,20 @@ export class SDL {
     );
   }
 
+  serviceResourceStorageAttributes(attributes?: Record<string, any>) {
```

Comment:
There is a type in `./types` which is the same however would simplify things when we get to refactoring

```suggestion
  serviceResourceStorageAttributes(attributes?: v2StorageAttributes) {
```